conversations/document_based.py

```diff
@@ -25,47 +24,29 @@ def __init__(self):
         self.llm = OobaboogaLLM()
         self.text_splitter = CharacterTextSplitter(
             chunk_size=1000, chunk_overlap=0)
-        self.vector_store = Chroma()
-        self.prompt = PromptTemplate(
+        self.vector_store_docs = Chroma()
+        self.vector_store_convs = Chroma()
```

Comment:
this will assume we are storing conversations and documents in the same collection (currently called `brainchulo`). We could configure a different one; and we will eventually persist and cleanup conversations for training or creating new documents.